src/List/NonEmpty.elm

```diff
@@ -895,3 +895,18 @@ decode =
 encodeList : (a -> Value) -> NonEmpty a -> Value
 encodeList f =
     Encode.list f << toList
+
+
+{-| Partition a non empty list on some test. The first list contains all values
```

Comment:
This wording comes from core right? I find it awkward but I guess consistency makes sense.

Reply:
Exactly 😆